libraries/entities/src/EntityItem.cpp

```diff
@@ -2700,20 +2714,6 @@ void EntityItem::setLastEdited(quint64 lastEdited) {
     });
 }
 
-quint64 EntityItem::getLastBroadcast() const {
-    quint64 result;
-    withReadLock([&] {
-        result = _lastBroadcast;
-    });
-    return result;
-}
-
-void EntityItem::setLastBroadcast(quint64 lastBroadcast) {
-    withWriteLock([&] {
-        _lastBroadcast = lastBroadcast;
-    });
-}
-
```

Comment:
These are atomic operations that do not require locks.  I'm optimizing them to be inline.